Assets/StarterAssets/FirstPersonController/Scripts/FirstPersonController.cs

```diff
@@ -74,11 +73,13 @@ public class FirstPersonController : MonoBehaviour
 		private float _terminalVelocity = 53.0f;
         private float currentStamina;
         //private int currentJumps = 0;
-        private bool extraJump = false;
+        private bool extraJump = true;
         private float JumpTimer;
         private float FatigueTimer;
 		private bool sprinting = false;
 		private bool isFatigued = false;
+        private float fatigueRecharge = 0f;
+		private float _staminaRechargeMultiplier = 1f;
```

Comment:
Created a variable instead of directly using "1f" when multplying with "Time.deltaTime" for stamina recharge/decharge calculations